<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumismatistHandbook</class>
 <widget class="QMainWindow" name="NumismatistHandbook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>551</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>697</width>
    <height>551</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>697</width>
    <height>551</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Нумизмат</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>361</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="nameLayout">
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Наименование монеты</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameLineEdit">
       <property name="maxLength">
        <number>50</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="valueLayout">
     <item>
      <widget class="QLabel" name="valueLabel">
       <property name="text">
        <string>Номинал монеты</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="valueLineEdit">
       <property name="maxLength">
        <number>30</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="rarityCheckBox">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>270</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Раритет</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>300</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="numberLayout">
     <item>
      <widget class="QLabel" name="numberLabel">
       <property name="text">
        <string>Количество сохранившихся монет</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="numberSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>350</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="countryLayout">
     <item>
      <widget class="QLabel" name="countryLabel">
       <property name="text">
        <string>Государство, выпустившее монету</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="countryComboBox">
       <item>
        <property name="text">
         <string>Российская Империя</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>СССР</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Россия</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>410</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="qualitiesLayout">
     <item>
      <widget class="QLabel" name="qualitiesLabel">
       <property name="text">
        <string>Особенности</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="qualitiesLineEdit">
       <property name="maxLength">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>681</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item>
      <widget class="QPushButton" name="firstPushButton">
       <property name="text">
        <string>Выбрать запись №1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="secondPushButton">
       <property name="text">
        <string>Выбрать запись №2</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="showPushButton">
       <property name="text">
        <string>Показать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="savePushButton">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>299</width>
      <height>129</height>
     </rect>
    </property>
    <property name="title">
     <string>Год выпуска</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>299</width>
       <height>109</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="eraLayout">
      <item>
       <widget class="QLabel" name="eraLabel">
        <property name="text">
         <string>Эра</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="eraComboBox">
        <item>
         <property name="text">
          <string>До нашей эры</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Наша эра</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="centuryLayout">
        <item>
         <widget class="QLabel" name="centuryLabel">
          <property name="text">
           <string>Век</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="centurySpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="yearLayout">
        <item>
         <widget class="QLabel" name="yearLabel">
          <property name="text">
           <string>Год</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="yearSpinBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
